server/middlewares/validators/search.ts

```diff
@@ -41,9 +41,22 @@ const videoChannelsSearchValidator = [
   }
 ]
 
+const videoChannelsOwnSearchValidator = [
```

Comment:
Why `OwnSearch`?

Reply:
Because there's already a `videoChannelsSearchValidator` for public searches, which allows `searchTarget` as query parameter.

Reply:
Then `videoChannelsListSearchValidator`?

Reply:
?

Reply:
replaced